go-controller/pkg/cluster/gateway_init.go

```diff
@@ -5,11 +5,16 @@ package cluster
 import (
 	"fmt"
 	"net"
+	"regexp"
+	"strings"
 	"syscall"
 
 	"github.com/openvswitch/ovn-kubernetes/go-controller/pkg/config"
 	"github.com/openvswitch/ovn-kubernetes/go-controller/pkg/util"
+	"github.com/sirupsen/logrus"
```

Comment:
Separate other imported modules and ovn-kubernetes modules?

Reply:
I think we don't have a good style requirement here. I see that different file has different way this is done. For the time being, let us leave 'make lint' and 'make gofmt' to catch this - else, let us leave it as-is.